.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,2 @@
+- name: Run golangci-lint
```

Comment:
I'm not familiar with workflows being this short in nature -> doesn't this need a top level name and then a job defined?